myuw/templates/handlebars/card/welcome_card.html

```diff
@@ -0,0 +1,18 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "welcome_card" %}
+<div class="myuw-card">
+    <h3 class="myuw-card-title">
+        Welcome to MyUW, a student portal
+    </h3>
+
+    <div class="myuw-card-section last">
+        <p>MyUW is your go-to place for application status, UW resources for applicants, your account information, and much more.</p>
+        <h4 class="myuw-card-heading">In MyUW you can:</h4>
+	        <ul style="padding-left:20px">
```

Comment:
Not critical for this release, but we should probably standardize how we want bulleted lists to look and add that to the style. Adjusting the left padding on just this one seems strange to me.

Reply:
I couldn't find any other place with a bullet list, so I threw the first one here.  The bootstrap's left padding looked too big to me.